design-proposals/klip-47-add-source-stream-table-semantic.md

```diff
@@ -0,0 +1,122 @@
+# KLIP 47 - Add Source Stream/Table Semantic 
+
+**Author**: Boyang Chen (@boyang) | 
+**Release Target**: TBD | 
+**Status**: _In Discussion_ | 
+**Discussion**: https://github.com/confluentinc/ksql/pull/7474
+
+**tl;dr:** _Add support for source stream/table. This will give users a read-only view of data that 
+are populated by the upstream Kafka topic, making the data ownership more clear._
+           
+## Motivation and background
+
+Users of KsqlDB have no clear ownership definition around tables or streams they created. In a “normal” database, 
+when a user creates a table the DB creates the corresponding files on disk and owns the table. 
+User may write data into those tables via `INSERT INTO` statements and/or modify/delete existing data 
+via `UPDATE` and `DELETE` statements. However, the table will be not changed by any other means. 
+
+For ksqlDB, the setup is quite differently. Specifically, user could create the database based on 
+the topics stored in Kafka as the source of truth. The defined database could be mutable by 
```

Comment:
The source of truth would always be Kafka topics for ksqlDB -- the point is _who_ owns those topics, and who is writing into them, right? If there is an upstream CDC pipeline that populates the topic, the CDC producer "owns" the topic and ksqlDB should only read it, but not write into it.